sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -86,6 +86,24 @@ export interface SendOptions {
   abortSignal?: AbortSignalLike;
 }
 
+/**
+ * The set of options to configure the createBatch operation on the `Sender`.
+ */
+export interface BatchOptions {
+  /**
+   * @property
+   * A value that is hashed to produce a partition assignment.
+   * It guarantees that messages with the same partitionKey end up in the same partition.
+   * Specifying this will throw an error if the producer was created using a `paritionId`.
+   */
+  partitionKey?: string | null;
+  /**
+   * @property
+   * The maximum size allowed for the batch.
+   */
+  maxMessageSize?: number | null;
```

Comment:
I see`partitionKey` in `SendOptions` allows `null` as a value. Since this is an object that the customer passes in rather than positional parameters specified in a function, could these fields just be optional arguments and omit the `null` type?